src/js/directives/bsdate.js

```diff
@@ -14,7 +14,7 @@ angular.module('xeditable').directive('editableBsdate', ['editableDirectiveFacto
                  **/
                 this.parent.render.call(this);
 
-                var inputDatePicker = angular.element('<input type="text" class="form-control" ng-model="$data"/>');
+                var inputDatePicker = angular.element('<input type="text" class="form-control" data-ng-model="$data"/>');
```

Comment:
Why did you change `ng-model` to `data-ng-model`? 
I don't think it is necessary.